.deepsource.toml

```diff
@@ -0,0 +1,13 @@
+version = 1
+
+[[analyzers]]
+name = "javascript"
+enabled = true
+
+[[analyzers]]
+name = "docker"
+enabled = true
+
+[[analyzers]]
+name = "ruby"
+enabled = true
```

Comment:
What's this doing?

Reply:
I'll remove it, this was giving me some analytics for this github plugin thing, I'll remove it